### PR TITLE
Add ingressClassName in ingress chart

### PR DESCRIPTION
**Pull request by aliartiza75**
this PR will add ingressClassName in the ingress chart

### CODE REVIEW

**Comment by stakater-user** [comment]
@aliartiza75 Validation Successful

**Comment by stakater-user** [comment]
@aliartiza75 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/ee56c2ff1ebcb2e348281d656bf3b32ed224b9ac/checks) has Failed!

**Comment by aliartiza75** [comment]
@mustafaStakater Please review this PR

**Comment by hanzala1234** [comment]
@aliartiza75  please add entry in values.yaml and values-test.yaml as well

**Comment by hanzala1234** [comment]
@aliartiza75  please update README with changelog as well

**Comment by hanzala1234** [comment]
https://github.com/stakater-charts/application/blob/master/README.md#changelog

File: application/values.yaml [hunk 1]
```diff
@@ -298,6 +298,9 @@ service:
 ingress:
   enabled: false
   
+  # Name of the ingress class
+  ingressClassName: nginx
```
Comment (hanzala1234) [inline application/values.yaml:302]: by default it has to be empty

Comment (hanzala1234) [reply application/values.yaml:302]: in values-test it should be fine

Comment (aliartiza75) [reply application/values.yaml]: @hanzala1234 done

**Comment by aliartiza75** [comment]
@hanzala1234 thank you

**Comment by hanzala1234** [comment]
@aliartiza75  pipeline has filed due to some reasons but chart has been published. looking at pipeline

**Pull request merged**